src/gnome_abrt/views.py

```diff
@@ -528,6 +444,45 @@ def __init__(self, application, sources, controller):
 
         self.connect("key-press-event", self._on_key_press_event)
 
+        self._add_actions(application)
+
+
```

Comment:
afaik only one new line between functions

Reply:
Maybe. This just preserves the two newlines that were there before.

Reply:
oh ok